### DISCUSSION TITLE
could not determine local IPv4 address: "Network unreachable" - using localhost

### DISCUSSION THREAD

**Discussion by bugith** [jamulussoftware/jamulus#1139] [Technical Support & Help] [answered]
Hi
I see this message when looking server status, is there a problem here ?

systemctl status jamulus.service 
● jamulus.service - Jamulus jam server
   Loaded: loaded (/etc/systemd/system/jamulus.service; enabled; vendor preset: enabled)
   Active: active (running) since sam. 2021-02-20 09:24:58 CET; 1 weeks 0 days ago
 Main PID: 1103 (jamulus)
   CGroup: /system.slice/jamulus.service
           └─1103 /usr/local/bin/jamulus -s -n -p 12345 -u 5 -o My;Here;74

févr. 20 09:24:58 nux jamulus[1103]: - server mode chosen
févr. 20 09:24:58 nux jamulus[1103]: - no GUI mode chosen
févr. 20 09:24:58 nux jamulus[1103]: - selected port number: 12345
févr. 20 09:24:58 nux jamulus[1103]: - maximum number of channels: 5
févr. 20 09:24:58 nux jamulus[1103]: - server info: My;Here;74
févr. 20 09:24:58 nux jamulus[1103]: could not determine local IPv4 address: "Network unreachable" - using localhost
févr. 20 09:24:58 nux jamulus[1103]: Recording state not initialised
févr. 20 09:24:58 nux jamulus[1103]:  *** Jamulus, Version 3.5.9git
févr. 20 09:24:58 nux jamulus[1103]:  *** Internet Jam Session Software
févr. 20 09:24:58 nux jamulus[1103]:  *** Released under the GNU General Public License (GPL)

**Comment by hoffie** [comment, marked as answer]
If you run a server in a private network, register it to a dictionary server (central server) in the Internet, but expect private clients to connect to your server, then this is relevant. This error would mean that those clients would connect using the public IP instead, which may not work or is at least unnecessary.

If this is not the case for you, you can probably ignore this message. The next Jamulus version will use a different mechanism for doing that. I suspect that it might avoid this message in your case.

I suspect that your machine does not have Internet access. Is that possible?

**Reply by bugith** [reply]
Thanks for quick reply man. I sudo systemctl restart jamulus.service and the message isn't here anymore. I guess this could happen on machine startup, if network is not ready when jamulus starts (maybe a dhcp delay, so i'll try static IP instead of dhcp reservation, and check).
Bye bye
